python/awaazde/apis.py

```diff
@@ -82,10 +82,13 @@ def upload(self, file_path):
 
         return self._client.post(upload_url, **data)
 
-    def create_bulk(self, data):
+    def create_bulk(self, data, transform_using_template=False, **kwargs):
         """
         This will create new object
         """
         bulk_url = self.url + "create_bulk/"
-        data = {'json': data}
-        return self._client.post(bulk_url, **self._append_headers(data))
\ No newline at end of file
+        # We need to pop send "transform_using_template" as a separate parameter other than "data"
```

Comment:
@ashwini-ad please remove these comments and please add a method parameter explainatation.